### DISCUSSION TITLE
Project cannot be restored through the dashboard

### DISCUSSION THREAD

**Discussion by fmussari** [supabase/supabase#27201] [General]
Hi, one of me free projects has this message: "This project has been paused for over 90 days and cannot be restored through the dashboard."
I would like to know if I can restore it and how. Thanks

**Comment by GaryAustin1** [comment]
Contact support.

**Comment by Spreceriany** [comment]
Happened to me as well. I am contacting support if they can help.

**Reply by Bradley-Bird** [reply]
did they ever get back to you?

**Comment by Spreceriany** [comment]
Yes they did and successfully recovered the project

**Reply by YousefSaeed668** [reply]
they do it for u or help u to do it ?

**Comment by Spreceriany** [comment]
Did it for me after I asked

**Comment by dvdgd** [comment]
If I restore the backup, there's any way to have this project setup again?
